.jabbarc

```diff
@@ -0,0 +1,2 @@
+# https://github.com/shyiko/jabba
+jdk: adopt@1.8.0-222
```

Comment:
Can this be wildcarded to any JDK matching 1.8.* ? 

Does this enforce adopt JDK? Is it still working if a user has installed OpenJDK or Oracle?

Reply:
I updated the required version to be a wildcard, but `adopt@1.8.0-222` is somehow sorted above `@1.14` ... but other OpenJDK and Oracle versions work with this wilrcard.